src/openrct2/ride/Ride.cpp

```diff
@@ -4214,9 +4214,9 @@ static bool ride_check_start_and_end_is_station(CoordsXYE* input)
     {
         return false;
     }
-    ride->chairlift_bullwheel_location[1].x = trackFront.x >> 5;
-    ride->chairlift_bullwheel_location[1].y = trackFront.y >> 5;
-    ride->chairlift_bullwheel_z[1] = trackFront.element->base_height;
+    ride->ChairliftBullwheelLocation[1].x = trackFront.x >> 5;
```

Comment:
Create a CoordsXYZ from trackFront.x, trackFront.y and trackFront.element->GetBaseZ, then convert that to TileCoordsXYZ.